src/store/fileUploadStore.js

```diff
@@ -321,7 +321,7 @@ const actions = {
 				message: index === lastIndex && caption ? caption : '{file}',
 			}
 			// Add temporary messages (files) to the messages list
-			dispatch('addTemporaryMessage', temporaryMessage)
+			dispatch('addTemporaryMessage', { token: temporaryMessage.token, message: temporaryMessage })
```

Comment:
Here in `uploadFiles` you kept using token from message but for `retryUploadFiles` , you switched to `getToken` ? 
Pick one :p